README.md

```diff
@@ -46,6 +46,7 @@ jobs:
       with:
         assets: test_results/output.xml
         aviator_api_token: ${{ secrets.AVIATOR_API_TOKEN }}
+        assets_required: true  # NOTE: This is optional (the default value is false). If true, the action will fail if no assets are found.
```

Comment:
let's do default to true? That way this is proactively detecting it but users have a way to turn it off?